app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,13 @@
+from .dwarf import Dwarf
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(self, skill_level: int, nickname: str, favourite_dish: str):
+        super().__init__(nickname, favourite_dish)
```

Comment:
The order of parameters in the `super().__init__()` call is incorrect. It should match the order defined in the `Dwarf` class constructor. Swap `nickname` and `favourite_dish` in the `super().__init__()` call to `super().__init__(favourite_dish, nickname)`.